modules/apps/map/map-common/src/main/resources/META-INF/resources/js/MapBase.es.js

```diff
@@ -106,15 +106,26 @@ class MapBase extends State {
 				: {};
 
 		if (!geolocation.lat && !geolocation.lng) {
-			Liferay.Util.getGeolocation(
-				(lat, lng) => {
-					this._initializeLocation({lat, lng});
-				},
-				() => {
-					this.zoom = 2;
-					this._initializeLocation({lat: 0, lng: 0});
-				}
-			);
+			if (navigator.geolocation) {
+				navigator.geolocation.getCurrentPosition(
+					(position) => {
+						const {latitude, longitude} = position.coords;
+
+						this._initializeLocation({
+							lat: latitude,
+							lng: longitude,
+						});
+					},
+					() => {
+						this.zoom = 2;
+						this._initializeLocation({lat: 0, lng: 0});
```

Comment:
In this case, when having errors, we'll set the lat/lng to the [null island](https://en.wikipedia.org/wiki/Null_Island). What about throwning an openToast here containing the information that we are initializing the location with 0,0?

Reply:
> @diegonvs In this case, when having errors, we'll set the lat/lng to the [null island](https://en.wikipedia.org/wiki/Null_Island). What about throwning an openToast here containing the information that we are initializing the location with 0,0?

You're suggesting to open a pop-up when a location isn't enabled by the user that notifies them the default location has been set to 0, 0?

The openToast is the little pop up on the bottom left, correct?

Reply:
> You're suggesting to open a pop-up when a location isn't enabled by the user that notifies them the default location has been set to 0, 0?

Yes, this is just a random thought that I would ask someone from design about this but it's just my opinion

Reply:
I see, let's not introduce any new functionality to this as it's not in the scope of the epic, but it's a neat idea 👍